### PR TITLE
Added better installation method

### PR DESCRIPTION
**Pull request by amilajack**
This is probably what most users intend to do

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,15 @@
 
 ## Install
 
-```console
-$ npm install eslint-loader
+```bash
```
Comment (MoOx) [inline README.md]: This is not correct. bash is for bash script, and this is a console snippet. Please just add --save-dev, but not even sure it's mandatory...

File: README.md [hunk 2]
```diff
@@ -4,15 +4,15 @@
...
+$ npm install eslint-loader --save-dev
 ```
 
 ## Usage
 
 In your webpack configuration
 
-```javascript
+```js
```
Comment (MoOx) [inline README.md]: why is this better?

Comment (amilajack) [inline README.md]: I saw that line [79 had the line](https://github.com/MoOx/eslint-loader/edit/master/README.md#L79) had that style so i confirmed the other styles to it.

**Comment by amilajack** [comment]
Fixed

**Comment by MoOx** [comment]
Thank.

**Pull request merged**